src/pages/gd/Bridge/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { memo, useCallback, useMemo } from 'react'
+import { useSwitchNetwork } from '@gooddollar/web3sdk-v2'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+import { useApplicationTheme } from 'state/application/hooks'
+import {
+    KimaTransactionWidget,
+    KimaProvider,
+    FontSizeOptions,
+    ColorModeOptions,
+    ModeOptions,
+    DAppOptions,
+} from '@kimafinance/kima-transaction-widget'
+import '@kimafinance/kima-transaction-widget/dist/index.css'
+
+const Bridge = memo(() => {
+    const { library } = useActiveWeb3React()
+    const [theme] = useApplicationTheme()
+    const { switchNetwork } = useSwitchNetwork()
+
+    const successHandler = useCallback(() => {
+        console.log('Kima bridge success:')
+    }, [])
+
+    const errorHandler = useCallback((e) => {
+        console.log('Kima bridge error:', e?.message, e)
+    }, [])
+
+    const switchChainHandler = useCallback(
+        async (chainId) => {
+            await switchNetwork(chainId)
+        },
+        [switchNetwork]
+    )
+
+    const options = useMemo(() => {
+        return {
+            theme: {
+                colorMode: theme === 'dark' ? ColorModeOptions.dark : ColorModeOptions.light,
+                fontSize: FontSizeOptions.medium,
+                fontFamily: 'Roboto',
+                backgroundColorDark: 'rgb(21, 26, 48)',
+            },
+            mode: ModeOptions.bridge,
+            dAppOption: DAppOptions.G$,
+            kimaBackendUrl: 'https://gooddollar-beta.kima.finance',
+            kimaNodeProviderQuery: 'https://api_testnet.kima.finance',
+            provider: library,
+            autoConnect: false,
+            compliantOption: false,
+            helpURL: 'https://t.me/GoodDollarX',
+        }
+    }, [theme, library])
```

Comment:
```suggestion
    }), [theme, library])
```

Reply:
It seems I fixed it already but recovered while adding console.log there.
It's done now.